### ISSUE TITLE
cookie vs. wp-cron.php

### ISSUE THREAD

**Issue by thabbs** [qtranslate/qtranslate-xt#779]
It looks like that qtranslate-xt tries to set the language cookie even when invoked by wp-cron.php. If there is no reason to do so it shouldn't try.

This warning is logged every time wp-cron is running:
```
Cannot modify header information - headers already sent in
[...]/wp-content/plugins/qtranslate-xt/qtranslate_core.php on line 463
```

**Comment by herrvigg** [comment]
Which version are you using? The `set_cookie` call should be on line 473 not 463. The `qtranxf_can_redirect` is checking `wp_doing_cron` so i don't know, need more investigation. Check in debug mode, i also added some asserts that can help to understand.

**Comment by herrvigg** [comment]
@thabbs I just updated Wiki [Troubbleshooting](https://github.com/qtranslate/qtranslate-xt/wiki/Troubleshooting). Maybe you could enable the WP_DEBUG_LOG to better see what's going on. I can't find the problem without any information from your side.

**Comment by herrvigg** [comment]
@thabbs i can't reproduce this, so i close it for now. Feel free to re-open when you can provide more information.

**Comment by herrvigg** [comment]
@thabbs This should be solved by #1114, just merged in master.
